frontend/src/views/Create.tsx

```diff
@@ -1,9 +1,40 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { createRoom, Room } from '../api/Room';
+import { PrimaryButton } from '../components/core/Button';
+import { isError } from '../api/Error';
+import ErrorMessage from '../components/core/Error';
+import Loading from '../components/core/Loading';
 
 function CreateGamePage() {
+  const history = useHistory();
+  const [error, setError] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  const createNewRoom = useCallback(() => {
+    const redirectToGame = (room: Room) => {
+      history.push('/game', { room });
+    };
+
+    setLoading(true);
+    createRoom()
+      .then((res) => {
+        setLoading(false);
+        if (isError(res)) {
+          setError(res.message);
+        } else {
+          redirectToGame(res as Room);
+        }
+      });
+  }, [history]);
+
   return (
     <div>
-      Create game page
+      {error ? <ErrorMessage message={error} /> : null}
+      {loading ? <Loading /> : null}
+      <PrimaryButton onClick={createNewRoom}>
+        Create New Room
+      </PrimaryButton>
```

Comment:
If you did this, you would likely update the design slightly on this page to make the loading or error text bigger (as there may be nothing else on this page). Or, you could just not have this page at all, and display loading and error messages on the homepage. I'm open to any of these ideas -- thoughts?

Reply:
I got rid of the page so now create new room on the landing page just takes you directly to the game page. At some point though we might need to re-add the create page though, or maybe not; I can also imagine that creating a new game would just take you to the waiting room and that's where hosts might be able to customize their rooms.

Reply:
> I can also imagine that creating a new game would just take you to the waiting room and that's where hosts might be able to customize their rooms.
That's what I was thinking; I think it would reduce the barrier to starting the game, which is what we want. This could also allow participants to see or edit the game settings themselves.